### ISSUE TITLE
NSExpression function is forbidden

### ISSUE THREAD

**Issue by losdwind** [aws-amplify/amplify-ios-maplibre#22]
### Describe the bug

Refer to this patch [Fix for "NSExpression function is forbidden" #411](https://github.com/maplibre/maplibre-gl-native/pull/411)
It also happens in my environment using Amplify Geo Plugin.
`
2022-09-09 19:59:48.259147+0800 Feunote[20138:3421145] [general] NSPredicate: Use of 'mgl_interpolate:withCurveType:parameters:stops:' as an NSExpression function is forbidden.
2022-09-09 19:59:48.259592+0800 Feunote[20138:3421145] [general] NSPredicate: Use of 'mgl_step:from:stops:' as an NSExpression function is forbidden.
`

### Steps To Reproduce

```swift
Steps to reproduce the behavior:
1. Add amplify-ios-maplibre
2. import AmplifyMapLibreUI
3. add AMLMapView() anywhere under ContentView
4. See error
```

### Expected behavior

Would like to have a synchronous fix.

### Amplify MapLibre Adapter Version

1.0.0

### Swift version

5.6.1

### Xcode version

13.4.1(13F100)

### Is this a regression?

No

### Device

iPhone 11 Pro Simulator

### iOS Version

iOS 15.5

**Comment by atierian** [comment]
Thanks for opening this issue. We'll investigate using an updated version that includes this fix. Is this causing build / compiler errors for you?

**Comment by losdwind** [comment]
> Thanks for opening this issue. We'll investigate using an updated version that includes this fix. Is this causing build / compiler errors for you?

Yes, It makes my code cannot compile on iOS 15.5 and iOS16. I have to turn to Apple MapKit Library as a workaround.

**Comment by ruisebas** [comment]
Fixed in [2.1.0](https://github.com/aws-amplify/amplify-ios-maplibre/releases/tag/2.1.0)
